ccp.h

```diff
@@ -83,7 +83,7 @@ struct ccp_connection {
     // the index of this array element
     u16 index;
 
-    u32 last_create_msg_sent;
+    u64 last_create_msg_sent;
```

Comment:
what is this for? maybe we should add an explanatory comment, since ccp.h is supposed to be the public api.

Reply:
see comment description of bbdd0f5

Reply:
ah nevermind, it's not really described there, i'll add a little more detail..

`last_create_msg_sent` is set to `datapath->now()`, which returns a `u64`

Reply:
no, the fix is fine, I was more asking why this field is there. it doesn't seem to need to be public, so maybe we should move it into the ccp_priv state?